### PR TITLE
Hotfix/docker build permissions

### PR DESCRIPTION
**Pull request by kytomaki**
Mounting docker volume proved to be problematic. This sidesteps it by building the rpm on the docker image build phase. Then instantiating the container and copying the rpm from there.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -117,19 +121,33 @@ vendor: glide.lock | $(BASE) ; $(info $(M) retrieving dependencies…)
...
+		bin/keepsake
 
 .PHONY: amzn
-amzn: docker/Dockerfile
-	docker run -v `pwd`:/rpmbuild -t amazonlinux:$(AMAZONLINUX_VERSION)-Development bash -c ' make -C /rpmbuild package'
+amzn: $(RPM_PACKAGE)
+
+$(RPM_PACKAGE): Dockerfile
+	docker run \
+		--name $(AMAZONLINUX_VERSION)-Developmen-$(PID) \
```
Comment (jaakkoo) [inline Makefile:138]: Developmen

File: Makefile [hunk 2]
```diff
@@ -11,6 +11,10 @@ TESTPKGS = $(shell env GOPATH=$(GOPATH) $(GO) list -f '{{ if or .TestGoFiles .XT
 
 AMAZONLINUX_VERSION := 2017.09
 GLIDE_VERSION := v0.13.1
+PID := $$$$
```
Comment (jaakkoo) [inline Makefile:14]: ???

**Review by jaakkoo — CHANGES_REQUESTED** [review]
Typos like "Developmen" .. 

So do we need to build Docker image to build a rpm?

**Comment by kytomaki** [comment]
This shouldn't be needed any more.

**Pull request closed without merging**